cmd/rot13.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/getlantern/rot13"
+)
+
+func die(format string, args ...interface{}) {
+	fmt.Fprintf(os.Stderr, format, args...)

```

Comment:
This is missing an `os.Exit(1)`, right?

Reply:
Yep, not sure where it ran off to. Added.